pkg/controller/account/account_controller.go

```diff
@@ -333,21 +335,18 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 	}
 
 	if (currentAcctInstance.Status.State == "") && (currentAcctInstance.Status.Claimed == false) {
-		// before doing anything make sure we are not over the limit if we are just error
-		accountTotal, err := TotalAwsAccounts(awsSetupClient)
-		if err != nil {
-			reqLogger.Info("Failed to get AWS account total from AWS api", "Error", err.Error())
-			return reconcile.Result{}, err
-		}
-
-		if accountTotal >= awsLimit {
-			reqLogger.Error(ErrAwsAccountLimitExceeded, "AWS Account limit reached", "Account Total", accountTotal)
-			return reconcile.Result{}, ErrAwsAccountLimitExceeded
-		}
 
 		var awsAccountID string
 
 		if currentAcctInstance.Spec.AwsAccountID == "" {
+			// before doing anything make sure we are not over the limit if we are just error
+			accountTotal := totalaccountwatcher.TotalAccountWatcher.Total
+			if accountTotal >= AwsLimit {
+				reqLogger.Error(ErrAwsAccountLimitExceeded, "AWS Account limit reached", "Account Total", accountTotal)
+				return reconcile.Result{}, ErrAwsAccountLimitExceeded
+			} else {
+				reqLogger.Info("Limit not reached")
```

Comment:
We don't need to log that the limit isn't reached

Reply:
Ack